app/services/ITSAStatusService.scala

```diff
@@ -32,10 +31,10 @@ import scala.concurrent.{ExecutionContext, Future}
 class ITSAStatusService @Inject()(itsaStatusConnector: ITSAStatusConnector,
                                   dateService: DateService,
                                   implicit val appConfig: FrontendAppConfig) extends FeatureSwitching {
-  private def getITSAStatusDetail(taxYear: TaxYear, futureYears: Boolean, history: Boolean)
-                                 (implicit user: MtdItUser[_], hc: HeaderCarrier, ec: ExecutionContext): Future[List[ITSAStatusResponseModel]] = {
+  private def getITSAStatusDetail(taxYear: TaxYear, futureYears: Boolean, history: Boolean, nino: String)
```

Comment:
I think we are limiting ourself here, not sure why we have to switch from user: MtdItUser[_] to use nino ? 
anyway, this is something you can confirm with Solar team; buts its fine with me